src/python/pants/backend/python/target_types.py

```diff
@@ -384,6 +368,22 @@ class PythonTests(Target):
     core_fields = (*COMMON_PYTHON_FIELDS, PythonTestsSources, PythonCoverage, PythonTestsTimeout)
 
 
+# -----------------------------------------------------------------------------------------------
+# `python_library` target
+# -----------------------------------------------------------------------------------------------
+
+
+class PythonLibrarySources(PythonSources):
+    default = ("*.py",) + tuple(f"!{pat}" for pat in PythonTestsSources.default)
```

Comment:
Nit
```suggestion
    default = ("*.py", *(f"!{pat}" for pat in PythonTestsSources.default))
```

Reply:
I had it that way at first, but I didn't like the way it looked because of the confusion with the "*.py" preceding it. Two completely unrelated meanings of `*` so close together...

Reply:
Oh, good point. Sounds good.